solutions/usaco-1011.mdx

```diff
@@ -15,6 +15,48 @@ find the maximum over all right triangles all in $\mathcal{O}(N^3)$ time.
 
 <LanguageSection>
  
+
+<PySection>
+
+```python
+from itertools import permutations
+# using itertools to generate all permutations
+
+fin = open('triangles.in', 'r')
+
+N = int(fin.readline().strip())
+points = []
+
+for _ in range(N):
+	points.append([int(p) for p in fin.readline().split()])
+
+pers = list(permutations(points, 3))
+# generating all permutations of size 3, since a triangle consists of 3 points.
+
+area = 0
+
+for x in pers:
+
+	if x[0][1] == x[1][1] and x[1][0] == x[2][0]:
+		# checks if one of the sides is parallel to the x-axis and another is parallel to the y-axis.
```

Comment:
```suggestion
		# checks if one of the sides is parallel to the x-axis
		# and another is parallel to the y-axis.
```
this line is too long; split into two lines

Reply:
got it.